stream.go

```diff
@@ -1483,6 +1483,9 @@ type ServerStream interface {
 	// It is safe to have a goroutine calling SendMsg and another goroutine
 	// calling RecvMsg on the same stream at the same time, but it is not safe
 	// to call SendMsg on the same stream in different goroutines.
+	//
+	// The message should not be modified after calling SendMsg. Both the
+	// tracing library and stats.Handlers handles the message lazily.
```

Comment:
It would be nicer to stay consistent with the existing language in the doc string. How about:

```golang
// It is not safe to modify the message after calling SendMsg. Tracing libraries
// and stats handlers may use the message lazily.
```

Reply:
For sure!. Updated. PTAL. Thanks for the review!